### PR TITLE
re design navigation

### PR DESCRIPTION
**Pull request by nonoakij**
## Reference(s)
before|after
-|-
viron plus_dashboard_endpoints_ (1)|localhost_8000_dashboard_endpoints_

## Checklist
- [x] [changeset file(s)](https://github.com/changesets/changesets/blob/main/docs/adding-a-changeset.md) included

### CODE REVIEW

File: packages/app/src/components/navigation/internalPages/index.tsx [hunk 1]
```diff
@@ -0,0 +1,60 @@
...
+    icon: <BarsOutLineIcon className="w=em" />,
+  },
+];
+
+type Props = BaseProps;
+const InternalPages: React.FC<Props> = ({ className = '', on }) => {
+  const { t } = useTranslation();
+
+  const { originalPath } = useI18n();
+  console.log(originalPath, on);
```
Comment (ejithon) [inline packages/app/src/components/navigation/internalPages/index.tsx:34]: nits: plz remove this line.

Comment (nonoakij) [reply packages/app/src/components/navigation/internalPages/index.tsx]: [7c96903](https://github.com/cam-inc/viron/pull/674/commits/7c96903fb65162daa864db9bfb4126f98c23051b)

File: packages/app/src/components/navigation/internalPages/index.tsx [hunk 2]
```diff
@@ -0,0 +1,60 @@
...
+}[] = [
+  {
+    to: INTERNAL_PAGE_PATHS.ROOT,
+    label: 'internalPagePaths.root',
+    icon: <HomeOutlineIcon className="w=em" />,
+  },
+  {
+    to: INTERNAL_PAGE_PATHS.ENDPOINTS,
+    label: 'internalPagePaths.endpoints',
+    icon: <BarsOutLineIcon className="w=em" />,
```
Comment (ejithon) [inline packages/app/src/components/navigation/internalPages/index.tsx:25]: Perhaps the classname w=em is a typo?

Comment (nonoakij) [reply packages/app/src/components/navigation/internalPages/index.tsx]: [be7d2ce](https://github.com/cam-inc/viron/pull/674/commits/be7d2cebf1694963cd79458bb7a8028ff2f4feb8)

**Pull request merged**